Source/WTF/wtf/SequesteredImmortalHeap.h

```diff
@@ -50,6 +51,168 @@
 
 namespace WTF {
 
+struct GranuleHeader {
```

Comment:
How about using `DoublyLinkedListNode`?

Reply:
I've done so locally but it seems to have introduced some bugs. At the very least DoublyLinkedList doesn't have move semantics, but even after accounting for that I'm still getting corruptions -- need to investigate further